src/modules/renderGraphiQL.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Mostly taken straight from express-graphql, so see their licence
+ * (https://github.com/graphql/express-graphql/blob/master/LICENSE)
+ */
+
+// TODO: in the future, build the GraphiQL app on the server, so it does not
+// depend on any CDN and can be run offline.
+
+export type GraphiQLData = {
```

Comment:
One more thing I'd like to add is the ability to set headers, but I'm not 100% sure how. One hacky way would be to let people provide a fetch function that we then serialize. What do you think?